expression/builtin_math_vec.go

```diff
@@ -223,3 +225,47 @@ func (b *builtinRoundDecSig) vecEvalDecimal(input *chunk.Chunk, result *chunk.Co
 func (b *builtinRoundDecSig) vectorized() bool {
 	return true
 }
+
+func (b *builtinPowSig) vecEvalReal(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf1, err := b.bufAllocator.get(types.ETReal, n)
```

Comment:
we can reuse `result` to store one of the results of the two parameters, thus the vectors involved in the calculation can be reduced from 3 to 2. Maybe the performance can be improved by this tiny optimization.